confgenerator/confgenerator_test.go

```diff
@@ -30,7 +30,10 @@ import (
 const (
 	validTestdataDir   = "testdata/valid"
 	invalidTestdataDir = "testdata/invalid"
-	defaultConfig      = "default-config.yaml"
+	// Relative path to the ops-agent/confgenerator folder.
+	defaultUserConfPath = "default-config.yaml"
+	// Test name inside ops-agent/confgenerator/testdata/valid/{linux|windows}
```

Comment:
I would not mention the `ops-agent/` prefix…

Reply:
Removed